### PR TITLE
data:Chinwenma.json

### PR DESCRIPTION
**Pull request by Chinwenma**

### CODE REVIEW

File: data/Chinwenma.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Chinwenma Okorie",
+  "bio": " **FrontEnd Developer** | Open Source enthusiast | Blogger | Tech enthusiast",
+  "tags":[ " Open Source, NextJs, ReactJs,  JavaScript, Tailwind, Html5, CSS3"],
```
Comment (kumarsonsoff3) [inline data/Chinwenma.json:4]: Please fix the array syntax here.

```suggestion
  "tags":[ " Open Source", "NextJs", "ReactJs",  "JavaScript", "Tailwind", "Html5", "CSS3"],
```

File: data/Chinwenma.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        {
+      "name": " Check out my GitHub Profile",
+      "url": "https://github.com/Chinwenma",
+      "icon": "FaGit"
+    },
+          {
+      "name": "Let's Connect on LinkedIn",
+      "url": "https://github.com/Chinwenma",
+      "icon": "FaLinkedin"
+    },
```
Comment (kumarsonsoff3) [inline data/Chinwenma.json:40]: Please remove this trailing comma here! 

```suggestion
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Chinwenma.json (hunk 1 above)
Comment (Chinwenma) [reply data/Chinwenma.json:4]: ok thanks

File: data/Chinwenma.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      {
+       "icon": "FaGit",
+      "url": "https://github.com/Chinwenma"
+    },
+          {
+      "icon": "FaLinkedin",
+       "url": "https://github.com/Chinwenma"
+
+    },
```
Comment (kumarsonsoff3) [inline data/Chinwenma.json:18]: Also, remove this trailing comma! 

```suggestion
    }
```

File: data/Chinwenma.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      "url": "https://twitter.com/Ethel_Omma"
+    }
+    { "icon": "FaGithub",
+      "url": "https://github.com/Chinwenma"
+    }
+     { "icon": "FaLinkedin",
+       "url": "https://www.linkedin.com/in/chinwenma-okorie-5414511a6?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3Bib%2FYeFWESsiAGxMfYOHqFQ%3D%3D"
+     }
+  ],
+
```
Comment (kumarsonsoff3) [inline data/Chinwenma.json:23]: You've unfortunately removed all the required commas

```suggestion
    "socials": [
    { "icon": "FaTwitter", 
      "url": "https://twitter.com/Ethel_Omma"
    },
    { "icon": "FaGithub",
      "url": "https://github.com/Chinwenma"
    },
     { "icon": "FaLinkedin",
       "url": "https://www.linkedin.com/in/chinwenma-okorie-5414511a6?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3Bib%2FYeFWESsiAGxMfYOHqFQ%3D%3D"
     }
  ],

```

File: data/Chinwenma.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      "url": "https://twitter.com/Ethel_Omma",
+      "icon": "FaTwitter"
+    }
+        {
+      "name": " Check out my GitHub Profile",
+      "url": "https://github.com/Chinwenma",
+      "icon": "FaGit"
+    }
+          {
+      "name": "Let's Connect on LinkedIn",
```
Comment (kumarsonsoff3) [inline data/Chinwenma.json:41]: Same here as well.

```suggestion
      "icon": "FaBlog"
    },
    {
      "name": " Follow me on Twitter",
      "url": "https://twitter.com/Ethel_Omma",
      "icon": "FaTwitter"
    },
        {
      "name": " Check out my GitHub Profile",
      "url": "https://github.com/Chinwenma",
      "icon": "FaGit"
    },
          {
      "name": "Let's Connect on LinkedIn",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**